staging/src/k8s.io/apiserver/pkg/admission/plugin/webhook/testing/webhook_server.go

```diff
@@ -160,3 +168,38 @@ func webhookHandler(w http.ResponseWriter, r *http.Request) {
 		http.NotFound(w, r)
 	}
 }
+
+func ClockSteppingWebhookHandler(fakeClock *clock.FakeClock) func(http.ResponseWriter, *http.Request) {
+	return func(w http.ResponseWriter, r *http.Request) {
+		path := r.URL.Path
+		delay, err := strconv.ParseInt(path[strings.LastIndex(path, "/")+1:], 0, 64)
+		if err != nil {
+			http.NotFound(w, r)
+		}
+		if strings.HasPrefix(path, "/allow/") {
```

Comment:
nit:

switch {
case strings.HasPrefix(path, "/allow"):
   ...
case strings.HasPrefix(path, "'/disallow"):
   ...
default:
  http.NotFound(w, r)
}

Reply:
Changed this a bit, hopefully its a bit more clear now.